api/resource_groups.go

```diff
@@ -69,29 +70,57 @@ const (
 	OciResourceGroup
 )
 
+// query templates
+var (
+	NoneResourceGroupQueryTemplate string = ""
+	//go:embed _templates/resource_groups/aws.json
+	AwsResourceGroupQueryTemplate string
+	//go:embed _templates/resource_groups/azure.json
+	AzureResourceGroupQueryTemplate string
+	//go:embed _templates/resource_groups/container.json
+	ContainerResourceGroupQueryTemplate string
+	//go:embed _templates/resource_groups/gcp.json
+	GcpResourceGroupQueryTemplate string
+	//go:embed _templates/resource_groups/machine.json
+	MachineResourceGroupQueryTemplate   string
+	LwAccountResourceGroupQueryTemplate string = ""
+	//go:embed _templates/resource_groups/oci.json
+	OciResourceGroupQueryTemplate string
+)
+
+type resourceGroupContext struct {
+	typ           string
```

Comment:
Can we name this something other `typ`? `resourceGroupType`?

Reply:
If we must...

Reply:
Fixed.